log/log.go

```diff
@@ -0,0 +1,192 @@
+// Copyright © 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package log
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+const (
+	// LevelLogError specified the log level as: ERROR
+	LevelLogError = 1
+
+	// LevelWarning specified the log level as: WARNING
+	LevelWarning = 2
+
+	// LevelLogInfo specified the log level as: INFO
+	LevelLogInfo = 3
+
+	// LevelLogDebug specified the log level as: DEBUG
+	LevelLogDebug = 4
+
+	// LevelLogVerbose specified the log level as: VERBOSE
+	// This is the same as Debug, but without the repeat filtering
+	LevelLogVerbose = 5
+
+	// Mixer is use as a cmdTag when mixer command is logging error
+	Mixer = "MIXER"
+	// Dnf is use as a cmdTag to when dnf command is logging error
+	Dnf = "DNF"
+	// Rpm is use as a cmdTag when rpm is logging error
+	Rpm = "RPM"
+	// Rpm2Archive is use as a cmdTag when rpm2archive is logging error
+	Rpm2Archive = "RPM2ARCHIVE"
+	// Tar is use as a cmdTag when tar is logging error
+	Tar = "TAR"
+	// BsDiff is use as a cmdTag when bsDiff is logging error
+	BsDiff = "BSDIFF"
+)
+
+var (
+	level      = LevelLogInfo
+	levelMap   = map[int]string{}
+	fileHandle *os.File
+	logging    = false
+	lineLast   string
+	lineCount  int
+	cmdMap     = map[string]bool{}
+)
+
+func init() {
+	levelMap[LevelLogError] = "LevelLogError"
```

Comment:
Update the descriptions as:
```
	levelMap[LogLevelError] = "ERROR"
	levelMap[LogLevelWarning] = "WARNING"
	levelMap[LogLevelInfo] = "INFO"
	levelMap[LogLevelDebug] = "DEBUG"
	levelMap[LogLevelVerbose] = "VERBOSE"
```

Reply:
as package name is log, it was giving me lint error for LogLevelError, variable name starting with log